integration-tests/gatsby-source-wordpress/test-fns/data-resolution.js

```diff
@@ -6,9 +6,13 @@ const {
   default: fetchGraphql,
 } = require("gatsby-source-wordpress/dist/utils/fetch-graphql")
 
+const isWarmCache = process.env.WARM_CACHE
```

Comment:
Tiny nitpick :sweat_smile: could you move this down below the `require`s?

Reply:
for sure 😄